src/backend/api/reservations.js

```diff
@@ -0,0 +1,118 @@
+import express from "express";
+const router = express.Router();
+import knex from "../database.js";
+
+// /api/reservations	GET	Returns all reservations
+router.get('/', async (req, res) => {
+    try {
+        const reservations = await knex.select().from('reservation');
+        if (reservations) {
+            res.json(reservations);
+        }else {
+            res.send('No reservation found.')
+        }
+    } catch (error){
+        console.error(error);
+    }
+})
+
+// /api/reservations	POST	Adds a new reservation to the database
+router.post('/', async (req, res) => {
+    try {
+        const {id, number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email} = req.body;
+        const newReservation = await knex('reservation').insert({
+            id,
+            number_of_guests,
+            meal_id,
+            created_date,
+            contact_phonenumber,
+            contact_name,
+            contact_email
+        });
+        if (newReservation){
+          res.status(201).json(newReservation)
+        }
+    } catch (error) {
+        console.error(error);
+        res.status(500).json({ messagge: "Internal server error"})
+    }
+});
+
+// My new reservations:
+// {
+//   "id": 5,
+//   "number_of_guests": 2,
+//   "meal_id": 3,
+//   "created_date": "2024-04-07 23:00:00",
+//   "contact_phonenumber": "1234569",
+//   "contact_name": "Mette Pedersen",
+//   "contact_email": "mettep@gmail.com"
+// }
+
+// {
+//   "id": 6,
+//   "number_of_guests": 1,
+//   "meal_id": 1,
+//   "created_date": "2024-04-06T22:00:00.000Z",
+//   "contact_phonenumber": "1234565",
+//   "contact_name": "Henrik Henriksen",
+//   "contact_email": "henrikh@gmail.com"
+// }
+
+// {
+//   "id": 7,
+//   "number_of_guests": 1,
+//   "meal_id": 1,
+//   "created_date": "2024-04-06T22:00:00.000Z",
+//   "contact_phonenumber": "1234565",
+//   "contact_name": "Henrik Henriksen",
+//   "contact_email": "henrikh@gmail.com"
+// }
```

Comment:
please remove unused code, for references you should always either documentation/readme file or in the future Types (Typescript, proptypes or even JSCDoc) :)

Reply:
oh okay I didn't think about that🙈